src/sentry/options/__init__.py

```diff
@@ -63,6 +63,7 @@ def impl_locally() -> AbstractOptionsStore:
 set = default_manager.set
 delete = default_manager.delete
 register = default_manager.register
+unregister = default_manager.unregister
```

Comment:
What is this for ?

Reply:
I believe this is exposing the `unregister` function outside of the module. This appears to be the `public` interface of options and we're anticipating using the unregister function.

Reply:
Originally I assumed that delete was not deleting properly (it would only set value to none, but key remained in database). I tested in devserver so that could be why I thought delete was implemented as such. Still not sure if this is the case.

Reply:
As per offline discussion, unregistering an option is different from unsetting it. I don't think the tool should ever unregister anything as options are registered in code via `src/sentry/options/defaults.py`